scss/modules/_breadcrumbs.scss

```diff
@@ -0,0 +1,202 @@
+        &:first-of-type {
+          border-bottom: 1px solid $alto-grey;
+
+          @media only screen and (min-width: $breakpoint-large) {
+            border-bottom: 0;
+          }
+        }
+
+        a {
+          font-size: 1em;
+          display: block;
+          box-sizing: border-box;
+          width: 100%;
+          margin-right: 0;
+          padding: 8px 10px 0;
+          text-decoration: none;
+          color: $cool-grey;
+
+          @media only screen and (min-width: $breakpoint-large) {
+            font-size: .875em;
+          }
+        }
+
+        .breadcrumb-link {
+          display: none;
+
+          @media only screen and (min-width: $breakpoint-large) {
+            display: block;
+          }
+        }
+
+        .breadcrumb-toggle {
+
+          @media only screen and (min-width: $breakpoint-large) {
+            display: none;
+          }
+
+          .breadcrumb-toggle__close,
+          .breadcrumb-toggle__open {
+
+            &:after {
+              position: relative;
+              top: 0;
+              right: 0;
+              float: right;
+              width: 18px;
+              height: 18px;
+              margin-top: -6px;
+              margin-right: -5px;
+              padding: 10px;
+              background-image: url('#{$asset-path}img/patterns/nav-down-arrow.svg');
+              background-repeat: no-repeat;
+              background-position: center center;
+              background-size: 18px;
+              display: block;
+
+              @media only screen and (min-width: $breakpoint-large) {
+                display: none;
+              }
+            }
+          }
+
+          .breadcrumb-toggle__close {
+            display: none;
+          }
+
+          .breadcrumb-toggle__open {
+            display: block;
+          }
+        }
+        // scss-lint:enable NestingDepth
+      }
+    }
+
+    .second-level-nav {
+      display: none;
+      overflow: hidden;
+      width: 100%;
+      margin: 0;
+      padding-top: 10px;
+      padding-bottom: 10px;
+      border-bottom: 1px solid $alto-grey;
+
+      @media only screen and (min-width: $breakpoint-large) {
+        display: block;
+        float: none;
+        margin-bottom: 0;
+        padding: 0;
+        width: auto;
+        border-bottom: 0;
+      }
+
+      li {
+        float: left;
+        width: 50%;
+        margin: 0;
+
+        @media only screen and (min-width: $breakpoint-large) {
+          width: auto;
+          margin-right: 15px;
+          margin-bottom: 5px;
+        }
+
+        // scss-lint:disable NestingDepth
+        a {
+          font-size: .875em;
+          display: block;
+          box-sizing: border-box;
+          width: 100%;
+          height: 100%;
+          padding: 10px;
+          color: $cool-grey;
+
+
+          &.active {
+            font-weight: 500;
+          }
+
+          @media only screen and (min-width: $breakpoint-large) {
+            padding: 8px 10px 0;
```

Comment:
The links look closer to the top then bottom. I think if this is set to 10px 10px 0 it vertically centered.